README.md

```diff
@@ -38,6 +38,10 @@ Grab your post-its. Ready? Brainstorm. mindwendel helps you to easily brainstorm
   ```ah
   mix ecto.setup
   ```
+  Add seeds with
+  ```
```

Comment:
Please add sh in order to enable consistent source code highlighting in markdown.